frontend/src/features/projects/dispose/ProjectDisposeLayout.tsx

```diff
@@ -177,6 +191,7 @@ const ProjectDisposeLayout = ({ match, location }: { match: Match; location: Loc
                   onSave={() => onSave(formikRef)}
                   onNext={onNext}
                   saveDisabled={currentStatus.code === DisposeWorkflowStatus.Approval}
+                  nextDisabled={rationaleRequired}
```

Comment:
`rationaleRequired` only disables the button on save but the schema already kind of covers the functionality that this is trying to achieve.  this just makes it so the user can't physically click the button if they save the form with the exemption checkbox set to true and no rationale. may not be necessary